### DISCUSSION TITLE
✅ 🔗 Option to offset the Image Carousel slides in both sides (Duplicate)

### DISCUSSION THREAD

**Discussion by nibfreelancer** [elementor/elementor#13489] [Feature Request]
Hello everyone.
I am using a default image carousel in ELEMENTOR, but I would like to use a centered mode, as can be seen in the following link
https://hado-official.com/en/about/ (HADO PLAYER'S GUIDE)

Could you guide me please
Thanks!

**Comment by nicholaszein** [comment]
Hello!

We have great news! 🙌 

### 📢 We're happy to announce that this feature was released with the new nested Carousel (Pro) and Loop Carousel (Pro) widgets in Elementor Pro v3.15! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
